Add note about known issue on release 2.6.2 (#110)
rancher/stackstate-product-docs: docs/latest/modules/en/pages/setup/release-notes/v2.6.2.adoc

== Release Notes: {stackstate-product-name} Helm Chart v2.6.2

== Known issues

> * This release contains 2 version of the Open Telemetry stackpack using the same logo. On-hover over the stackpack you can see the name. When installing or upgrading use the stackpack named "Open Telemetry" and don't install the "Open Telemetry 2.0" stackpack. The latter doesn't provide any functionality at the moment and will be removed in the next release.

Reviewer: ```suggestion
* This release contains 2 versions of the Open Telemetry stackpack using the same logo. Hover over the stackpack to see the name. When installing or upgrading, use the stackpack named "Open Telemetry" and do not install the "Open Telemetry 2.0" stackpack. The latter does not provide any functionality at the moment and will be removed in the next release.
```
